icu4c/source/data/Makefile.in

```diff
@@ -198,7 +198,7 @@ ifneq ($(ENABLE_STATIC),)
 ifeq ($(PKGDATA_MODE),dll)
 # For MinGW, do we want the DLL to go in the bin location?
```

Comment:
I wonder if it might be worth updating this comment. Maybe it should say something that for static builds, it is put in the libdir.

Reply:
Changed this comment to clarify target ([commit](https://github.com/unicode-org/icu/pull/1733/commits/a881ab95e57164d78744ad0ee570941b382e61ec))